### PR TITLE
Hotkeys: Add "Toggle USB Keyboard" hotkey

### PR DESCRIPTION
**Pull request by spycrab**
[Closes issue #10813](https://bugs.dolphin-emu.org/issues/10813).

### CODE REVIEW

**Comment by JMC47** [comment]
Seems to work for me, immediately activates the option.  Keyboard is working on the internet channel.  Haven't tested anything else.

**Comment by Just-Harry** [comment]
I tried this on Animal Crossing: Let's go to the City (City Folk).

|Hotkey used|Game started with USB Keyboard disabled|Game started with USB Keyboard enabled|
|-|-|-|
|Scroll-lock|Worked|Worked|
|Grave|Worked|Worked|
|K|Worked|Worked|

However, the very first time I enabled the USB keyboard I got an `Unable to resolve read address` error, and the very first time I disabled the USB keyboard [I got the same error](https://image.ibb.co/kzYMJz/2018_08_02_17_26_03.png), with a different address.
After that the errors didn't occur, unless I pressed the hotkey in quick succession.

I assume some kind of log would be required for debugging, but I'm not sure as to how I should configure the log.

---

Also, this closes [issue #10813](https://bugs.dolphin-emu.org/issues/10813), not issue #11314.

**Comment by JMC47** [comment]
That's probably unrelated to this hotkey.

File: Source/Core/Core/HotkeyManager.cpp [hunk 1]
```diff
@@ -271,7 +272,7 @@ constexpr std::array<HotkeyGroupInfo, NUM_HOTKEY_GROUPS> s_groups_info = {
      {_trans("Stepping"), HK_STEP, HK_SKIP},
      {_trans("Program Counter"), HK_SHOW_PC, HK_SET_PC},
      {_trans("Breakpoint"), HK_BP_TOGGLE, HK_MBP_ADD},
-     {_trans("Wii"), HK_TRIGGER_SYNC_BUTTON, HK_BALANCEBOARD_CONNECT},
+     {_trans("Wii"), HK_TRIGGER_SYNC_BUTTON, HK_TOGGLE_USB_KEYBOARD},
```
[the remark this answers is not in the document]
Comment (spycrab) [reply Source/Core/Core/HotkeyManager.cpp]: This only describes the range of items in the group. So all hotkeys from ``HK_TRIGGER_SYNC_BUTTON`` to ``HK_TOGGLE_USB_KEYBOARD`` will be included.

As ``HK_BALANCEBOARD_CONNECT`` is between the two it also will be included.

**Pull request merged**